roles/deploy_bmh/template/bmh.yml.j2

```diff
@@ -25,3 +25,6 @@ spec:
   rootDeviceHints:
     {{}}: {{ node_data['root_device_hint'] }}
 {% endif                                                                             %}
+{%- if 'nmstate' in node_data %}
+  preprovisioningNetworkDataName: {{ node_name }}-netconfig-secret
```

Comment:
I believe this is wrong. It would need to be `{{ node_name }}-nmstate-secret`

Depending on https://github.com/openstack-k8s-operators/ci-framework/pull/1437/files#diff-9c7c344f6baca959deed26179a04746fdc0036f9d3e9944061c034c02efbd3b2R5

Reply:
Agree, fixing in a followup PR.